src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -49,6 +49,7 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
   viewPort: any = '';
   public showSpinner = false;
   isManualRefresh = false;
+  servicesToShow = ['northbound', 'southbound', 'dispatcher', 'notification', 'management', 'bucketstorage'];
```

Comment:
rename servicesToShow => externalServices

Reply:
'northbound' and 'southbound' services are not externalServices. Due to which I didn't named it as `externalServices `.